WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -324,8 +335,39 @@ public void loggedInViaUsernamePassword(ArrayList<Integer> oldSitesIds) {
     }
 
     @Override
-    public void help() {
-        ToastUtils.showToast(this, "Help is not implemented yet.");
+    public void helpEmailScreen(String email) {
+        launchHelpshift(null, email,
+                HelpshiftHelper.chooseHelpshiftLoginTag(getLoginMode() == LoginMode.JETPACK_STATS, true));
```

Comment:
I see. I wasn't really sure how the modes are decided. Thanks for the clarification!